src/styles/variables.less

```diff
@@ -1,39 +1,62 @@
-// Base
-@borderRadius: 4px;
-
 // font properties
```

Comment:
I'd like to see some better grouping on this file. Something [ala bootstrap's approach](https://github.com/twbs/bootstrap/blob/d9a64a0281556240813de5bad7cd7ef2720b7a79/less/buttons.less#L1-L7)